glovo/src/test/java/org/example/glovo/service/OrderServiceTest.java

```diff
@@ -0,0 +1,200 @@
+package org.example.glovo.service;
+
+import org.example.glovo.entity.OrderEntity;
+import org.example.glovo.entity.ProductEntity;
+import org.example.glovo.exception.OrderNotFoundException;
+import org.example.glovo.exception.ProductNotFoundException;
+import org.example.glovo.mapper.OrderMapper;
+import org.example.glovo.model.Order;
+import org.example.glovo.model.Product;
+import org.example.glovo.repository.OrderRepository;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import java.util.*;
+
+import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.anyLong;
+import static org.mockito.Mockito.*;
+
```

Comment:
I suggest adding a post-conditioning after every test and\or after all tests so that the tests do not affect each other